subsys/bluetooth/host/conn.c

```diff
@@ -1643,6 +1646,12 @@ void bt_conn_set_state(struct bt_conn *conn, bt_conn_state_t state)
 			/* Cancel Connection Update if it is pending */
 			if (conn->type == BT_CONN_TYPE_LE) {
 				k_delayed_work_cancel(&conn->update_work);
+#if defined(CONFIG_BT_REMOTE_VERSION)
+	if (atomic_test_and_clear_bit(conn->flags,
```

Comment:
The indentation isn't right here - it should be the same as all other code in the same block, regardless of the #ifdef being there or not.

Reply:
Yeah, one problem. If the "if" is to match the indentation and `BT_CONN_PENDING_VERSION_INFO` is to match the `atomic_test_and_clear_bit()` the line is over 80 characters.

Reply:
80 is not the hard rule - it'll cause a warning but shouldn't prevent merging. It does give a good hint to look at overall code design though, and in this case it seems the sane thing to do would be to refactor the `bt_conn_set_state()` function into smaller helper functions for each state. Anyway, that's something for a separate PR. Main point is that exceeding 80 characters is a lesser evil than not following the indentation level.

Reply:
should be fixed now